app/dashboard/router.py

```diff
@@ -26,25 +26,35 @@
 import django_filters.rest_framework
 from bounty_requests.models import BountyRequest
 from kudos.models import KudosTransfer, Token
+from rest_flex_fields import FlexFieldsModelSerializer
 from rest_framework import routers, serializers, viewsets
 from rest_framework.pagination import PageNumberPagination
-from rest_flex_fields import FlexFieldsModelSerializer
 from retail.helpers import get_ip
 from townsquare.models import Comment, Favorite, Flag, Like, PinnedPost
 
-
 from .models import (
     Activity, Bounty, BountyFulfillment, BountyInvites, HackathonEvent, HackathonProject, Interest, Profile,
-    ProfileSerializer, SearchHistory, TribeMember, UserDirectory,
+    SearchHistory, TribeMember,
 )
 from .tasks import increment_view_count
 
 logger = logging.getLogger(__name__)
 
 
+class ProfileSerializer(FlexFieldsModelSerializer):
```

Comment:
notice this name is used already on models

Reply:
i removed it from models